tests/basic/upgrade_cluster_test.go

```diff
@@ -89,6 +89,18 @@ var _ = Describe("{UpgradeCluster}", func() {
 					log.Warnf("PDB validation skipped. Current Px-Operator version: [%s], minimum required: [%s]. Error: [%v].", opver, PDBValidationMinOpVersion, err)
 				}
 
+				// validate volume quorum during upgrade
+				if opver.GreaterThanOrEqual(ParallelUpgradeVersion) {
+					var vQourumError error
```

Comment:
Do we report is vQourumError is not nil?

Reply:
added the validation of error